controllers/commonservice_controller.go

```diff
@@ -117,14 +124,19 @@ func (r *CommonServiceReconciler) ReconcileMasterCR(instance *apiv3.CommonServic
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
-	if err := r.updateOperandConfig(newConfigs); err != nil {
+	isChanged, err := r.updateOperandConfig(newConfigs)
+	if err != nil {
 		if err := r.updatePhase(instance, CRFailed); err != nil {
 			klog.Error(err)
 		}
 		klog.Errorf("Fail to reconcile %s/%s: %v", instance.Namespace, instance.Name, err)
 		return ctrl.Result{}, err
 	}
 
+	if !isChanged {
+		r.Recorder.Event(instance, "Normal", "Noeffect", fmt.Sprintf("No update, existing CommonService CR has larger profile %s/%s", instance.Namespace, instance.Name))
```

Comment:
This message is not very clear to me.
I assume it should be "No update, resource sizings in the OperandConfig %s/%s are larger than the profile from CommonService CR %s/%s"